### ISSUE TITLE
[Feature-Request]Support for custom resolutions OR re-Definition of area to compare

### ISSUE THREAD

**Issue by DevParapalli** [josephp27/ReduxAL#2]

**Comment by DevParapalli** [comment]
Do I change the values in line 24 && 25 of file detect_gun.py for the correct capture?
are the values from top left?
do i need to resize the present images?

**Comment by josephp27** [comment]
Exactly. Lines 24 and 25 are what you are going to want to change if your display is different.

The values are from the top left, which start at 0,0

You may need to resize the current images saved in data. You might be able to get away with not creating all new images. MSE has to take in 2 images that are the exact width and height. If you create new images, you could keep them the same size and compare them with your images captured on line 24&25 in detect_gun.py. I imagine the error returned might be a little higher but that's okay.

**Comment by panikajo** [comment]
![image](https://user-images.githubusercontent.com/4964185/55223188-51fffd80-5216-11e9-955c-08d913adde70.png)
that it's normal? But not working for me :(

**Comment by josephp27** [comment]
The true or false you see is a simple log to verify when you're clicking the mouse or not. There is a weird issue I have noticed where sometimes things will freeze. (This is due to the hooking libraries used. I attempted to mitigate such freezes by spawning threads, which helps most of the time.) When starting the program, I suggest not clicking anywhere in the prompt, but instead having your first click be in the apex window. 

When you scroll, you'll also see a number and list show up. This is the gun detection system logging.

TL;DR:
1. After starting do not click in the cmd or powershell window. Have the first click be in the apex legends window to avoid and hanging problems. 
2. If you are hanging, quit the program by pressing the "L" key
3. Whenever you scroll, logging will show for the guns, displaying a number and a gun, if there is one

**Comment by josephp27** [comment]
Closing as no response from OP
